app/controllers/discover.js

```diff
@@ -97,13 +99,15 @@ export default Ember.Controller.extend({
                 source.type = 'elastic-search-result';
                 source.workType = source['@type'];
                 source.abstract = source.description;
+                source.subjects = source.subjects.map(function(each) {return {text: each};});
                 source.providers = source.sources.map(item => ({name: item}));
                 source.contributors = source.contributors.map(function(contributor) {
                     return {
                         users: {
-                            familyName: contributor.family_name,
-                            givenName: contributor.given_name,
-                            id: contributor['@id']
+                            familyName: contributor
+                            // familyName: contributor.family_name,
```

Comment:
Why are these lines commented out? Did the layout of fields returning from SHARE change?